mlir/include/air/Util/Util.h

```diff
@@ -60,6 +60,10 @@ DmaMemcpyNdOp getAIRDmaInBlock(mlir::Block *block);
 
 // Get channel declaration through channel symbol
 ChannelOp getChannelDeclarationThroughSymbol(ChannelInterface op);
+// Get ChannelPutOp from ChannelOp
+ChannelPutOp getChannelPutOpThroughSymbol(ChannelOp channel);
+// Get ChannelGetOp from ChannelOp
+ChannelGetOp getChannelGetOpThroughSymbol(ChannelOp channel);
```

Comment:
There will be multiple gets or puts in the broadcast case. Can we make this API return multiple ops from the start?

Reply:
Yes I agree--this is something that I've been working on. Will make a PR on this soon.